arch/risc-v/src/common/riscv_mmu.c

```diff
@@ -0,0 +1,129 @@
+/****************************************************************************
+ * arch/risc-v/src/rv64gc/riscv_mmu.c
```

Comment:
Change rv64gc to common and, how about rename this file to riscv_mmu_sv39.c ?

Reply:
Ah my bad, missed this. Will fix the path.

However, I would like to keep the current name, as the driver framework is generic and is designed to work with any SvXX implementation (although Sv39 is the only one implemented now).

Reply:
Great, and we can develop the driver for sv32/sv48 on qemu rv32/rv64 in future.